makefile

```
@@ -101,6 +111,27 @@ format:
 tools:
 	go get -v $(GOTOOLS)
 
+.PHONY: cleveldb rocksdb
+cleveldb:
+	@if [ ! -e $(CLEVELDB_DIR) ]; then         \
+		sh contrib/get_cleveldb.sh;        \
+	fi
+	@if [ ! -e $(CLEVELDB_DIR)/libcleveldb.a ]; then   \
+		cd $(CLEVELDB_DIR);                        \
+		mkdir build;                               \
+		cd build;                                  \
+		cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLEVELDB_BUILD_TESTS=OFF -DLEVELDB_BUILD_BENCHMARKS=OFF ..; \
+		make;                                      \
```

Let's do it CMake way.
```suggestion
		cmake -S . -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLEVELDB_BUILD_TESTS=OFF -DLEVELDB_BUILD_BENCHMARKS=OFF; \
		cmake --build build;                                      \
```